apps/Deskreen/install-32

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+mkdir .Deskreen || error 'Failed to make dir ".Deskreen"!'
+cd .Deskreen || error 'Failed to cd into .Deskreen!'
+#Download deskreen-1.0.1-armhf.deb from Discord
+wget https://raw.githubusercontent.com/Pi-Apps-Coders/files/main/Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to download "Deskreen-1.0.1-armv7l.AppImage.zip" from Mirror!'
+unzip -o Deskreen-1.0.1-armv7l.AppImage.zip || error 'Failed to unzip Deskreen-1.0.1-armv7l.AppImage.zip!'
+# Install
+chmod +x Deskreen-1.0.1-armv7l.AppImage || error 'Failed to make Deskreen-1.0.1-armv7l.AppImage executable!'
+echo "#!/bin/bash
+$HOME/.Deskreen/Deskreen-1.0.1-armv7l.AppImage" | sudo tee /usr/local/bin/deskreen || error 'Failed to make Deskreen terminal script!'
```

Comment:
Add `&>/dev/null` to hide the output, like this:
```
echo '#!/bin/bash
$HOME/.Deskreen/Deskreen-1.0.1-armv7l.AppImage' | sudo tee /usr/local/bin/deskreen &>/dev/null || error 'Failed to make Deskreen terminal script!'
```

Reply:
Ok. Thanks.